src/LayerPlan.cpp

```diff
@@ -741,10 +741,10 @@ void LayerPlan::addWall(ConstPolygonRef wall, int start_idx, const Settings& set
     });
     ewall.emplace_back(*wall.begin(), nominal_line_width, dummy_perimeter_id);
 
-    addWall(ewall, start_idx, settings, non_bridge_config, bridge_config, wall_0_wipe_dist, flow_ratio, always_retract, true, false);
+    addWall(ewall, start_idx, settings, non_bridge_config, bridge_config, wall_0_wipe_dist, flow_ratio, always_retract, true, false, false);
```

Comment:
That's nasty booltrap. I don't know why it wasn't caught in the previous reviews, but 3 bools without context in a row is getting a bit out of hand.

I suggest swapping em out for constexpr bool statements.